base/mpfr.jl

```diff
@@ -26,6 +26,9 @@ import Base.Math.lgamma_r
 
 import Base.FastMath.sincos_fast
 
+const MPFR_VERSION = VersionNumber(unsafe_string(ccall((:mpfr_get_version,:libmpfr), Ptr{Cchar}, ())))
+const MPFR_PATCHES = split(unsafe_string(ccall((:mpfr_get_patches,:libmpfr), Ptr{Cchar}, ())),' ')
```

Comment:
or we could move this ccall to run in the tests until we have a more important need for it?